modules/azurerm/API-Management-API-Operation-Multi-Method-With-Failover/versions.tf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+terraform {
+  required_version = ">= 0.14"
```

Comment:
Let's update this with `0.14.10`

Reply:
Updated with [b9ad4b2](https://github.com/wso2/azure-terraform-modules/pull/92/commits/b9ad4b2f2cf30100dd602fb713afa70ff97c9bfa)